src/inspect.rs

```diff
@@ -1,27 +1,65 @@
+use crate::{DockerConfig, Registry, Sources};
 use anyhow::{anyhow, Result};
 use mdcat::{ResourceAccess, TerminalCapabilities, TerminalSize};
+use policy_evaluator::policy_fetcher::oci_distribution::manifest::{OciImageManifest, OciManifest};
+use policy_evaluator::policy_fetcher::sigstore::cosign::{ClientBuilder, CosignCapabilities};
+use policy_evaluator::policy_fetcher::sigstore::registry::{Auth, ClientConfig};
```

Comment:
```suggestion
use policy_evaluator::policy_fetcher::{
  manifest::{OciImageManifest, OciManifest},
  sigstore::{
    cosign::{
      ClientBuilder,
      CosignCapabilities,
    },
    registry::{Auth, ClientConfig}
  },
};
```

Reply:
Non-blocking comment, it's a bit less repetitive. Feel free to ignore or change spacing etc...